packages/cli/src/util/get-args.ts

```diff
@@ -21,3 +24,36 @@ export default function getArgs<T extends Spec>(
     argv,
   });
 }
+
+type ParserOptions = {
+  permissive?: boolean;
+};
+
+/**
+ * Parses command line arguments.
+ * Automatically includes a number of common flags such as `--help`.
+ *
+ * This function is designed to replace getArgs
+ * and will live alongside getArgs until the migration is completed.
+ *
+ * It takes the same three arguments as getArgs with improved names: args, flagsSpecification, and parserOptions.
+ * It also changes the return type to be an object with two keys: {args, flags}
+ * `args` contains the positional arguments, previously returned under the key `_`
+ * `flags` contains the parsed options, previously these keys were mixed with the positional arguments
```

Comment:
You can use markdown in these comments. I spiced it up a bit in this example. This is what it looks like in VS Code:

<img width="829" alt="Screenshot 2024-04-10 at 2 22 31 PM" src="https://github.com/vercel/vercel/assets/41545/b5340238-f5ca-4dd2-a71a-15cb090e7f52">

```suggestion
 * Parses command line arguments.
 * Automatically includes a number of common flags such as `--help`.
 *
 * **Migrating from `getArgs`**
 *
 * This function is designed to replace `getArgs`
 * and will live alongside `getArgs` until the migration is completed.
 *
 * It takes the same three arguments as `getArgs` with improved names: `args`, `flagsSpecification`, and `parserOptions`.
 * It also changes the return type to be an object with two keys: `{args, flags}`
 *
 * - `args` was previously returned under the `_` key
 * - `flags` previously these keys were mixed with the positional arguments
```